v-next/hardhat-node-test-reporter/src/reporter.ts

```diff
@@ -149,21 +149,58 @@ export function hardhatTestReporter(
               }
             }
 
-            // If a suite/describe was already printed, we need to descrease
-            // the lastPrintedIndex, as we are removing it from the stack.
-            //
-            // If its nesting was 0, we print an empty line to separate top-level
-            // describes.
             if (event.data.nesting === 0) {
```

Comment:
Let's leave a comment here as it might not be 100% obvious what this branching means

```suggestion
            // If the nesting is 0, it means we're dealing with top-level entities
            if (event.data.nesting === 0) {
```